.evergreen/scripts/teardown-aws.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
We can merge some of these teardown scripts into `cleanup.sh`.

Reply:
Yeah, note how we handle it in the drivers-tools teardown: https://github.com/mongodb-labs/drivers-evergreen-tools/blob/aeb2d4af763ca5eab3bb29acb8038042d4268740/.evergreen/teardown.sh#L15